app/js/components/tabs/tabcomponents/conceptComponent.jsx

```diff
@@ -1,35 +1,153 @@
 import React, {Component} from 'react';
 
 class ConceptComponent extends Component {
-    componentDidMount(){}
+    constructor(props) {
+        super(props);
+        this.state = {
+            conceptsResults: [],
+            searchResults: [],
+            verbose: false
+        };
+        this.allConcepts = [];
+        this.foundConcepts = [];
+        this.verboseSearchResult = [];
+        this.searchConcept = this.searchConcept.bind(this);
+        this.checkVerbose = this.checkVerbose.bind(this);
+        this.loadConcepts = this.loadConcepts.bind(this);
+    }
+    /**
+     * Make call to /concepts to fetch all concepts and load into allConcepts array
+     * @param {*} props 
```

Comment:
@dkayiwa I just did, quite a large amount of data. We can make the search without the autocomplete feature if it will place a lot of load on the memory. What are your thoughts?